pkg/sqlite/sql.go

```diff
@@ -16,11 +16,6 @@ func selectAll(tableName string) string {
 	return "SELECT " + idColumn + " FROM " + tableName + " "
 }
 
-func distinctIDs(qb *queryBuilder, tableName string) {
```

Comment:
Why was this removed?

Reply:
Using distinct in postgresql requires you to select any column you plan to use, like if you do "order by".
SQLite doesn't have this limitation, but that meant that i either had to rewrite alot of sql, or simply removed distinctIDs.

Reply:
Ok, fair. We can add a TODO for the future if we want to optimize this

Reply:
I added this back